base_layer/core/src/base_node/consts.rs

```diff
@@ -23,6 +23,6 @@
 use std::time::Duration;
 
 /// The allocated waiting time for a request waiting for service responses from remote base nodes.
-pub const BASE_NODE_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(600);
+pub const BASE_NODE_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
I do not think this value should be changed at all, as too many things depend on this. Rather introduce a constant specifically for block sync that can be changed via the config file.